browser/ipfs/import/ipfs_import_controller.cc

```diff
@@ -294,10 +294,12 @@ void IpfsImportController::OnWebPageImportCompleted(
 
 void IpfsImportController::OnImportCompleted(const ipfs::ImportedData& data) {
   auto link = CreateAndCopyShareableLink(data);
+#if BUILDFLAG(ENABLE_IPFS_INTERNALS_WEBUI)
```

Comment:
This may look strange, but on desktop this flag is true, and on Android whole `IpfsImportController` class seems to be unused, but compiled. It looks it will be fixed with https://github.com/brave/brave-browser/issues/28060. 

My attempt to make it failed - https://github.com/brave/brave-core/compare/android_no_ipfs_webui_BAK01?expand=1#diff-87dbeae647f32aa0670c977a84d99be9862a63bf9cc1f0a8b191da86f1d1a21fR11 - with such change I could not open ipfs:// pages at all.